<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectionWidget</class>
 <widget class="QWidget" name="SectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Section Editor</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QGroupBox" name="headerGroup">
     <property name="title">
      <string>Section Header</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelLabel">
        <property name="text">
         <string>Label (e.g., Section A):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="labelEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="subtitleLabel">
        <property name="text">
         <string>Subtitle/Instructions:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="subtitleEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="questionsLayout">
     <property name="spacing">
      <number>10</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="controlsLayout">
     <item>
      <widget class="QPushButton" name="addQuestionButton">
       <property name="text">
        <string>Add Question</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>